src/dataset-builder/DatasetBuilder.test.ts

```diff
@@ -269,7 +269,8 @@ describe('DatasetBuilder', () => {
     const user = userEvent.setup();
     await initializeValidDatasetRequest(user);
     // Assert
-    expect(await screen.findByText('100 Participants in this dataset')).toBeTruthy();
+    expect(await screen.findByText('100')).toBeTruthy();
+    expect(await screen.findByText('participants in this dataset')).toBeTruthy();
```

Comment:
I expect that the funky way I'm handling the space between 100 & "participants in this dataset" is causing the original test to fail.

Reply:
It was -- this now passes.